src/typescript/azure-openapi-validator/rules/RequiredSystemDataInNewApiVersions.ts

```diff
@@ -10,7 +10,7 @@ export const RequiredSystemDataInNewApiVersions: string = "RequiredSystemDataInN
 rules.push({
   id: "R4009",
```

Comment:
Under **How to fix**, suggest to use common resource definition (by ARM team) from the following  that has systemdata 
specification\common-types\resource-management\v1\types.json

Reply:
Do you mean "How to fix" under docs @ https://github.com/Azure/azure-openapi-validator/tree/217995a157bcc01a1081efb2bc56b7465daa1585/docs?
There doesn't seem to be a file for this rule. Should I create one?

Reply:
sorry for the confusion. actually here [https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#r3025](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#r3025)

---
In reply to: [499902971](https://github.com/Azure/azure-openapi-validator/pull/225#discussion_r499902971) [](ancestors = 499902971)

Reply:
Thanks, good feedback, I will improve the how-to-fix
@josefree  About changing this rule to error, do you have any  suggestion ?